uiowa/uiowa
# Update dependencies.

Resolves #1281 
Resolves #1391 
Resolves #921
Resolves #1238
Resolves #853 
Resolves #141 

Configuration changes related to the following exist but were not updated:
- uiowa_lb_blocks (layout_builder_styles)
- entity_types_seminar (field_group, paragraphs)

Testing (dev)
- Log in to a dev site and test out any and all functionality.
  - https://uiowa.dev.drupal.uiowa.edu

Testing (local)
- Run `vagrant provision` locally to get PHP 7.3.
- Sync a few sites via `blt dsa` and ensure functionality (paragraphs, layout, default, custom) is as expected.
